lib/cc/engine/analyzers/ruby/main.rb

```diff
@@ -33,7 +34,37 @@ def overage(mass)
           end
 
           def process_file(file)
-            RubyParser.new.process(File.binread(file), file, TIMEOUT)
+            if File.extname(file) == ".erb"
+              process_erb file
+            else
+              RubyParser.new.process(File.binread(file), file, TIMEOUT)
+            end
+          end
+
+          def process_erb(file)
+            erb = File.read(file)
```

Comment:
Thanks for the catch, `binread` sounds good to me.

I pulled the code from `flay` and tested it out locally without an issue, and didn't realize the intention behind using `File.binread`. Making switch